env0/resource_environment.go

```diff
@@ -200,11 +206,17 @@ func setEnvironmentSchema(d *schema.ResourceData, environment client.Environment
 		return fmt.Errorf("schema resource data serialization failed: %v", err)
 	}
 
-	if environment.LatestDeploymentLog != (client.DeploymentLog{}) {
+	if environment.LatestDeploymentLog.BlueprintId != "" {
```

Comment:
The old `!=` can no longer work because json.RawMessage was added as field. It cannot be compared.